bulk-model-sync-gradle-test/ci.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/sh
+
+set -e
+
+cd "$(dirname "$0")"
+
+(
+  cd graph-lang-api
+  ./gradlew publishToMavenLocal --console=plain
```

Comment:
Here's some more potential for speed up in CI. If the build was performed by the GitHub Action with enable build cache, then recompiling / regenerating on every CI execution could be avoided.

Reply:
Seems like caching would still work like this, as the build action is used in a prior action, see [here](https://blog.gradle.org/gh-actions)